clap_derive/src/derives/args.rs

```diff
@@ -361,7 +361,7 @@ pub fn gen_augment(
 
 fn gen_arg_enum_possible_values(ty: &Type) -> TokenStream {
     quote_spanned! { ty.span()=>
-        .possible_values(&<#ty as clap::ArgEnum>::VARIANTS)
+        .possible_values(<#ty as clap::ArgEnum>::VARIANTS.into_iter().copied())
```

Comment:
Was the copied necessary? We could change the `ArgEnum` signature if needed.

Reply:
Without the copied we get `&&str`:
```
error[E0277]: the trait bound `ArgValue<'_>: From<&&str>` is not satisfied
   --> clap_derive/tests/arg_enum.rs:310:21
    |
310 |         arg: Option<ArgChoice>,
    |                     ^^^^^^^^^ the trait `From<&&str>` is not implemented for `ArgValue<'_>`
    |
    = help: the following implementations were found:
              <ArgValue<'help> as From<&'help str>>
    = note: required because of the requirements on the impl of `Into<ArgValue<'_>>` for `&&str`
    ```